src/js/models/StopDetails.js

```diff
@@ -89,13 +80,18 @@ StopDetails.prototype.parseResponse = function(res) {
 
         throw new Error(faultcode + ' ' + faultstring);
     }
+    Runs = res.query.results.Envelope.Body.Nextbus2Response.Runs.Run;
```

Comment:
I remember seeing that snippet around. While I never encountered it in testing, it's better to have it and not break it for now.